components/tag/css/index.scss

```diff
@@ -19,142 +19,112 @@
 
 // Default
 .rvo-tag--default {
+  --utrecht-icon-color: var(--rvo-tag-default-icon-color);
+
   background-color: var(--rvo-tag-default-background-color);
   color: var(--rvo-tag-default-color);
 }
 
-.rvo-tag--default .rvo-icon {
-  background-color: var(--rvo-tag-default-icon-color);
-}
-
 // Info
 .rvo-tag--info {
```

Comment:
De icons in Tag krijgen nog niet de juiste kleur, klopt dat?

Reply:
Ik zal deze ook nalopen.